libkbfs/folder_branch_ops.go

```diff
@@ -1824,6 +1825,8 @@ func (fbo *folderBranchOps) isRevisionConflict(err error) bool {
 
 func (fbo *folderBranchOps) finalizeMDWriteLocked(ctx context.Context,
 	lState *lockState, md *RootMetadata, bps *blockPutState, excl EXCL) (err error) {
+	fbo.log.CDebugf(ctx, "finalizeMDWriteLocked: excl=%s", excl)
```

Comment:
Please just log this in `CreateFile`, as part of the existing log line, so we don't spam the logs too badly.